packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -1050,7 +1050,7 @@ namespace Ifpack2 {
           SolveTridiagsDefaultModeAndAlgo<typename execution_space::memory_space>::
           recommended_team_size(blocksize, vector_length, internal_vector_length);
 
-        const local_ordinal_type num_teams = execution_space().concurrency() / (team_size * vector_length);
+        const local_ordinal_type num_teams = std::max(1, execution_space().concurrency() / (team_size * vector_length));
```

Comment:
That will still evaluate the division by zero - what about this?
```suggestion
        const local_ordinal_type teamvector_size = std::max(1, team_size * vector_length);
        const local_ordinal_type num_teams = execution_space().concurrency() / teamvector_size;
```